source/Hcl/HclStringLiteral.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace Octopus.Hcl
+{
+    public enum HclStringLiteralFormat
+    {
+        SingleLine = 0,
+        Heredoc = 1,
+        IndentedHeredoc = 2
+    }
+
+    public class HclStringLiteral
```

Comment:
I considered a class per format as `HeredocIdentifier` doesn't apply to `SingleLine`, but it seemed too much and too hard to switch the output format, which would be a common usage.